app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileScreen.kt

```diff
@@ -328,6 +336,46 @@ private fun OtherAccountItem(account: OtherAccount, onClick: (String) -> Unit =
     }
 }
 
+@Composable
+private fun DialogContent(
```

Comment:
I feel this file is becoming too big. Should we extract this component to a separate file annotated with `@PrivatePackage`?
Also to improve redability:
```suggestion
private fun UserStatusChangeDialogContent(
```